TanukiKit/Projects.swift

```diff
@@ -202,8 +207,8 @@ enum ProjectRouter: Router {
 
     var params: [String: Any] {
         switch self {
-        case .readAuthenticatedProjects(_, let page, let perPage):
-            return ["per_page": perPage as AnyObject, "page": page as AnyObject]
+        case .readAuthenticatedProjects(_, let page, let perPage, let archived, let visibility, let orderBy, let sort, let search, let simple):
+            return ["per_page": perPage as AnyObject, "page": page as AnyObject, "archived": archived as AnyObject, "visibility": visibility as AnyObject, "order_by": orderBy as AnyObject, "sort": sort as AnyObject, "search": search as AnyObject, "simple": simple as AnyObject]
```

Comment:
I think just having, e.g. `"archived": archived` should be enough as `String` type should conform to `Any`